src/sentry/static/sentry/app/components/avatar.jsx

```diff
@@ -1,7 +1,7 @@
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
-import $ from 'jquery';
+import qs from 'query-string';
```

Comment:
this starting causing problems with the avatars in the tests because we mock jquery.
one less component using jquery, and afaik they're equivalent behavior except for ordering